js/blocks/controls/image.js

```diff
@@ -68,15 +71,35 @@ const BlockLabImageControl = ( props, field, block ) => {
 			setAttributes( attr );
 		}
 
+		const uploadFiles = ( files ) => {
```

Comment:
This is copied verbatim [from below](https://github.com/getblocklab/block-lab/pull/260/commits/dc529cedb6a157b7c8aceacd3bfeb07a9f2371f6#diff-7bbb33042f7b94c527dc6a5bdb42c258L94).